drivers/storage/portworx/manifest/manifest.go

```diff
@@ -240,6 +246,14 @@ func fillDefaults(
 	if rel.Components.NodeWiper == "" {
 		rel.Components.NodeWiper = defaultNodeWiperImage
 	}
+	if rel.Components.DynamicPlugin == "" {
+		rel.Components.DynamicPlugin = defaultDynamicPluginImage
+	}
+
+	if rel.Components.DynamicPluginProxy == "" {
```

Comment:
Can you add UTs for these in manifest_test.go?

Reply:
aded these components in manifest_test.go